server/service/testing_utils.go

```diff
@@ -187,10 +191,27 @@ func RunServerForTestsWithDS(t *testing.T, ds fleet.Datastore, opts ...*TestServ
 	if len(opts) > 0 && opts[0].Logger != nil {
 		logger = opts[0].Logger
 	}
+	redisPool := redistest.SetupRedis(t, "error:", false, false, false)
```

Comment:
Does this mean that the MySQL tests will also require the `REDIS_TEST=1`? (Admittedly, I haven't looked too deeply into how those environmental variables are used.)